bundles/AdminBundle/config/pimcore/default.yaml

```
@@ -19,6 +19,135 @@ pimcore_admin:
     translations:
         path: "@PimcoreCoreBundle/translations"
 
+    user:
+        default_key_bindings:
+            save:
+                key: 'S'
+                ctrl: true
+            publish:
+                key: 'P'
+                ctrl: true
+                shift: true
+            unpublish:
+                key: 'U'
+                ctrl: true
+                shift: true
+            rename:
+                key: 'R'
+                alt: true
+                shift: true
+            refresh:
+                key: 't'
+            openAsset:
+                key: 'A'
+                ctrl: true
+                shift: true
+            openObject:
+                key: 'O'
+                ctrl: true
+                shift: true
+            openDocument:
```

```suggestion
            open_document:
```
I think we should use snake-case here for all the keys, as this is Symfony bast practice and we should be consistent here.

---

Yes this seems to be the right way. I Just did it to match the method names. Maybe we should add the method then as its  own property? I don't want to like fiddle around with all the keys to make proper function calls automatically. I would then set the key and the method as required
```yaml
            open_document:
                key: 'D'
                method(_name): openDocument
                ctrl: true
                shift: true
```

Something like this?

---

fixed it with adding `action` property